python/pyspark/ml/feature.py

```diff
@@ -86,7 +86,9 @@ class Binarizer(JavaTransformer, HasInputCol, HasOutputCol, JavaMLReadable, Java
     """
 
     threshold = Param(Params._dummy(), "threshold",
-                      "threshold in binary classification prediction, in range [0, 1]",
+                      "Param for threshold used to binarize continuous features.
+The features greater than the threshold will be binarized to 1.0. 
```

Comment:
I think this might not pass the style check either (`./dev/lint-python` will check it locally BTW). This also puts some whitespace in the string I think. Try this:

```
threshold = Param(Params._dummy(), "threshold",
                                 "Param for ... " + 
                                 "The features greater ...",
                                 ...)
```

Reply:
@srowen Just tried that - let me know if that works!

Reply:
Oh, did you git push to this branch?